ReactNativeClient/lib/joplin-renderer/MdToHtml/rules/video.js

```diff
@@ -0,0 +1,29 @@
+// LOGIC: Extract the domain name from the URL(attr: href of link_open)
+// & UUID of the youtube video and render iframe is domain is youtube
+// and the given youtube link has a valid UUID
+
+module.exports = function installRule(markdownIt) {
+
+	const defaultRender = markdownIt.renderer.rules.link_open;
+	const videoIDRegex = /^.*(youtu\.be\/|v\/|u\/\w\/|embed\/|watch\?v=|&v=)([^#&?]*).*/;
+	const key = /.*(youtube|youtu.be).*/im;
```

Comment:
Please could you change this to use only one regex. I think videoIDRegex should already give the ID so the `key` doesn't seem necessary.

Reply:
okay